examples/gzip-stream/bin/compress.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+"use strict";
+
+const { pipeline } = require("stream");
+const { Gzip, constants: { Z_BEST_COMPRESSION } } = require("zlib");
+
+const neon = require("..");
+const native = () => new Gzip({ level: Z_BEST_COMPRESSION });
+
+const compress = process.argv[2] === "built-in" ? native : neon;
+
+pipeline(
+    process.stdin,
+    native(),
```

Comment:
I assume this should call `compress()` instead?

```suggestion
    compress(),
```

Reply:
Yes, good catch. I'll probably also rename the other methods to `nativeCompress` and `neonCompress` to make it more clear.